src/Routing.js

```diff
@@ -39,55 +39,64 @@ export default function Routing({ appProps }) {
       Component: Overview,
       path: '/dashboard',
       allowedIf: userIsOfficerOrAdmin,
-      redirect: '/'
+      redirect: '/',
+      InAdminNavbar: true
```

Comment:
let's stick to using camelcase ([google suggests it](https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined))
```suggestion
      enableAdminNavbar: true
```